eoscompanion/eknservices_bridge.py

```diff
@@ -164,7 +164,7 @@ def _on_received_feed_query_results(_, result):
             return
 
         callback(None, [
-            ordered.model
+            ordered.get_property('model')
```

Comment:
Looks like this should be squashed with `eknservices: Add method to fetch the content feed from EosDiscoveryFeed`?